browser-interface/packages/shared/friends/reducer.ts

```diff
@@ -51,7 +51,7 @@ function reducePrivateMessaging(state: State, action: UpdatePrivateMessagingStat
 
 function reduceUpdateUserData(state: State, action: UpdateUserData) {
   const socialData = state.socialInfo[action.payload.socialId]
-  if (socialData && deepEquals(socialData, action.payload)) {
+  if (socialData && socialDeepEquals(socialData, action.payload)) {
```

Comment:
you could directly call `socialDeepEquals` here, right?
```suggestion
  if (socialDeepEquals(socialData, action.payload)) {
```